### PR TITLE
Added fix for IndexDB version not incrementing due to parallal execution of sql statements

### PR DESCRIPTION
**Pull request by Julias0**
Fix for #1641 

Inside 91indexeddb.js, 
```javascript
const request = indexedDB.open(ixdbid, found.version + 1);
```

Since PromiseAll triggers all of the statements at the same time, it causes version to get incremented only once even if multiple DDL statements are present. Have modified the behaviour to execute the queries sequentially which will prevent race conditions such as these from coming up

### CODE REVIEW

**Comment by mathiasrw** [comment]
This is such a good point. Thank you!

Any chance you can move all the functionality into the promiseAll function? Its doing what promiseAll should have been doing all the time.

File: test/browserTestRunner.js [hunk 1]
```diff
@@ -37,7 +37,7 @@
 		return results;
 	}
 
-	const testFiles = walkFiles(__dirname, /test\d{3}\.js$/, null, true, true);
+	const testFiles = walkFiles(__dirname, /test\d{3,4}\.js$/, null, true, true);
```
Comment (Julias0) [inline test/browserTestRunner.js:40]: The tests with 4-digit numbers were not being run inside the browser

**Comment by Julias0** [comment]
> This is such a good point. Thank you!
> 
> Any chance you can think about how we add a few tests for this?

Have added a new test to check this scenario @mathiasrw Lemme know if it works.

**Comment by mathiasrw** [comment]
I really like how precise you nailed this problem.

**Pull request merged**